modules/govuk_crawler/manifests/config.pp

```diff
@@ -0,0 +1,37 @@
+#
```

Comment:
Format the class docs to be consistent with other classes (i.e. add a `== Class: Govuk_crawler::Config` heading)